corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ACEReader.java

```diff
@@ -81,7 +81,7 @@
      * @throws Exception Exception thrown in-case of major failure.
      */
     public ACEReader(String aceCorpusHome, String[] sections, boolean is2004mode) throws Exception {
-        super(aceCorpusHome);
+        super(CorpusReaderConfigurator.buildResourceManager(aceCorpusHome));
```

Comment:
future work... I may actually get to it quite soon as I'm working with an ACE Relation Extraction package right now. Presently it ignores all the usual infrastructure requirements (based on old code, and reads annotations directly from ACE source files without parsing the xml...) so integrating the standard ACEReader is a likely step.